lib/explorer/backend/lazy_series.ex

```diff
@@ -39,6 +39,7 @@ defmodule Explorer.Backend.LazySeries do
     pow: 2,
     fill_missing_with_value: 2,
     fill_missing_with_strategy: 2,
+    format: 2,
```

Comment:
You still need to implement the callback in this module. I think you can look at `concat`, the implementation will be similar.

Reply:
Actually, `concat/2` is not a good example, because it does it two by two, we likely want to pass the whole list to Rust at once (it may be able to better optimize it). I will open up a separate issue for that.

Reply:
https://github.com/elixir-nx/explorer/pull/520/commits/d6f8564791942ee0db11584718e51e47add63916